syncthing_gtk/app.py

```diff
@@ -199,16 +199,16 @@ def do_command_line(self, cl):
 		else:
 			# Fallback for old GTK without option parsing
 			if "-h" in sys.argv or "--help" in sys.argv:
-				print "Usage:"
-				print "  %s [arguments]" % (sys.argv[0],)
-				print "Arguments:"
+				print("Usage:")
```

Comment:
except for maybe this.  the fallback probably isn't necessary anymore (although, idk what version CentOS 6/7 would be using, maybe it's old enough to still be necessary).